apps/examples/elf/wifiapp/wifiapp.c

```diff
@@ -42,6 +56,146 @@ int main(int argc, char **argv)
 	fprintf(stdout, "%15s: Hello, World!! on stdout\n", argv[0]);
 	fprintf(stderr, "%15s: Hello, World!! on stderr\n", argv[0]);
 
+#ifdef CONFIG_EXAMPLES_MPU_TEST
+	volatile uint32_t *address;
+	char ch;
+
+	printf("\nPress R - For Read Test\n");
+	printf("Press W - For Write Test\n");
+
+	ch = getchar();
+	if (ch == 'R' || ch == 'r') {
+		printf("\nPress A for app MPU test\n");
+		printf("Press K for kernel MPU test\n");
+
+		ch = getchar();
+
+		if (ch == 'A' || ch == 'a') {
+			address = APP_START_ADDR;
+
+			uint32_t dest;
+
+			printf("\n************************************************\n");
+			printf("************************************************\n");
+			printf("************************************************\n");
+			printf("* Test to verify protection of Micom app space *\n");
+			printf("************************************************\n");
+			printf("************************************************\n");
+			printf("************************************************\n");
+
+			printf("INFO: Read Micom app space: 0x%x\n", address);
```

Comment:
@kishore-sn Our final goal is obviously binary-separating Micom app and Wi-Fi app from kernel. However, here your sample apps (both Micom and Wi-Fi apps) doesn't include any micom and Wi-Fi functions, respectively. So, please, rename your test apps, for example, app1 and app2. 
In addition, real micom and Wi-Fi samples will be included soon. At this time, we would get confused.
What do you say?

Reply:
@davidfather 
I will change the app names in a different PR.